Rocket.Chat/API/Clients/MessagesClient.swift

```diff
@@ -58,7 +58,6 @@ struct MessagesClient: APIClient {
         }, errored: { error in
             switch error {
             case .version:
-                // TODO: Remove SendMessage Fallback + old methods after Rocket.Chat 1.0
```

Comment:
Let's just search for "case .version:" when we decide to remove fallbacks. That way we don't pollute with so many warnings.

Reply:
👍